APIBacklog/documentation/APIBacklog.md

```diff
@@ -3,51 +3,61 @@
+| OTP Validation | DTAG| 2022/09/30 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_NumberVerificationMS2FA_DeustcheTelekom.md) | Airtel, DTAG, GSMA-IDG, KDDI, Telefonica  | Approved (2022/10/13) | [OTPValidationAPI](https://github.com/camaraproject/OTPValidationAPI) | one-time-password-sms | [Link](https://github.com/camaraproject/OTPvalidationAPI/blob/main/documentation/API_documentation) | v0.5.0 | [YAML](https://github.com/camaraproject/OTPvalidationAPI/blob/main/code/API_definitions/one-time-password-sms.yaml) | N/A | N/A | N/A|N/A|%%%%%%%%%%%%%%%%<!---Carrier Billing CheckOut-->%%%%%%%%%%%%%%%%%%%%%
+| Carrier Billing Checkout | Telefonica | 2022/09/30 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_CarrierBillingCheckOut_Telefonica.md)| Airtel, GSMA-OGW, KDDI, Orange, Telefonica | Approved (2022/10/13) | [CarrierBillingCheckOut](https://github.com/camaraproject/CarrierBillingCheckOut)  | Carrier Billing Check Out| [Link](https://github.com/camaraproject/CarrierBillingCheckOut/blob/main/documentation/API_documentation) | v0.2.0 | [YAML](https://github.com/camaraproject/CarrierBillingCheckOut/blob/main/code/API_definitions/carrier_billing.yaml) | N/A| N/A | N/A|<!---Traffic Influence --->
+| Traffic Influence | Telecom Italia | 2022/11/02 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_Traffic%20Influence_TIM.md) | GSMA-OPAG, Telecom Italia | Approved (2022/11/10) | [EdgeCloud/TrafficInfluence](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Traffic%20Influence) | traffic-influence | [Link](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Traffic%20Influence) | v0.8.1| [YAML](https://github.com/camaraproject/EdgeCloud/blob/main/code/API_definitions/Traffic%20Influence/Traffic_Influence.yaml) | N/A | N/A | N/A | %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Blockchain Public Address-->%%%%%%%%%%%%%%%%%%%%%
+| Blockchain Public Address | Telefonica | 2023/01/19| [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_BlockchainPublicAddress_Telefonica.md)| Orange, STC, Telefonica| Approved (2023/06/15)  | [BlockchainPublicAddress](https://github.com/camaraproject/BlockchainPublicAddress)  | Blockchain Public Address | [Link](https://github.com/camaraproject/BlockchainPublicAddress/tree/main/documentation/API_documentation) | v0.1.0 | [YAML](https://github.com/camaraproject/BlockchainPublicAddress/blob/main/code/API_definitions/blockchain_public_address.yaml) | N/A| N/A | N/A |%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Identity and Consent Management-->%%%%%%%%%%%%%%%%%%%%%
```

Comment:
v0.1.0 is only "baseline contribution" and main branch is not any more the tagged v0.1.0